clients/java/src/test/java/com/google/myanmartools/TransliterateTest.java

```diff
@@ -568,7 +568,7 @@
           "ဝေသစ်လွင် about"
           },
     {  // #101 From case #134: Other
-      "ျပန္လမ္းမဲ့ကၽြန္းအပိုင္း၄",
+      "ျပန္လမ္းမဲ့ကြၽန္းအပိုင္း၄",
```

Comment:
I'm not sure why this test case is being changed. The old and new Z forms appear identical. Both should convert to the same Unicode string. Perhaps the test should include the two versions of the Z input, e.g, 
 "ျပန္လမ္းမဲ့ကၽြန္းအပိုင္း၄" +  "ျပန္လမ္းမဲ့ကြၽန္းအပိုင္း၄",
with the same Unicode string twice for expected output.

Reply:
yeah, both Z strings convert to the same Unicode string. But Unicode string also convert to only one Z string. In this case, "ျပန္လမ္းမဲ့ကြၽန္းအပိုင္း၄" should be the right output. if there are two versions of the Z input, the test will fail.

Reply:
Yes, this makes sense. Thx.